src/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -5,31 +5,91 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.IO.Tests
 {
-    public class NullCtorTests
+    public class NullTests
     {
+        [Theory]
+        [InlineData(20)]
+        public async static Task TestNullStream_Flush(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Flush();
+                await Stream.Null.FlushAsync();
+            }
+        }
+        
+        [Theory]
+        [InlineData(20)]
+        public static void TestNullStream_Dispose(int times)
+        {
+            for (int i = 0; i < times; i++)
+            {
+                Stream.Null.Dispose();
+            }
+        }
+        
+        [Theory]
+        [MemberData(nameof(NullStream_CopyToData))]
```

Comment:
This doesn't need to be a Theory; I'd suggest just using a MemoryStream in the body.